### PR TITLE
Send the full module name to dotnet test

### PR DESCRIPTION
**Pull request by mariam-abdulla**

### CODE REVIEW

File: src/Platform/Microsoft.Testing.Platform/IPC/IClient.cs [hunk 1]
```diff
@@ -3,12 +3,11 @@
...
 
-internal interface IClient : IDisposable
+internal interface IClient :
 #if NETCOREAPP
-#pragma warning disable SA1001 // Commas should be spaced correctly
-    , IAsyncDisposable
-#pragma warning restore SA1001 // Commas should be spaced correctly
+IAsyncDisposable,
 #endif
+IDisposable
```
Comment (Evangelink) [inline src/Platform/Microsoft.Testing.Platform/IPC/IClient.cs]:
```suggestion
    IAsyncDisposable,
#endif
    IDisposable
```

File: src/Platform/Microsoft.Testing.Platform/IPC/IServer.cs [hunk 2]
```diff
@@ -3,12 +3,11 @@
...
 
-internal interface IServer : INamedPipeBase, IDisposable
+internal interface IServer : INamedPipeBase,
 #if NETCOREAPP
-#pragma warning disable SA1001 // Commas should be spaced correctly
-    , IAsyncDisposable
-#pragma warning restore SA1001 // Commas should be spaced correctly
+IAsyncDisposable,
 #endif
+IDisposable
```
Comment (Evangelink) [inline src/Platform/Microsoft.Testing.Platform/IPC/IServer.cs]:
```suggestion
    IAsyncDisposable,
#endif
    IDisposable
```

File: src/Platform/Microsoft.Testing.Platform/IPC/NamedPipeBase.cs [hunk 3]
```diff
@@ -14,19 +14,19 @@ internal abstract class NamedPipeBase
...
 
     protected INamedPipeSerializer GetSerializer(int id)
         => _idSerializer.TryGetValue(id, out object? serializer)
             ? (INamedPipeSerializer)serializer
-            : throw new InvalidOperationException(string.Format(
+            : throw new ArgumentException(string.Format(
                 CultureInfo.InvariantCulture,
 #if PLATFORM_MSBUILD
-                "No serializer registered with id '{0}'",
+                "No serializer registered with ID '{0}'",
```
Comment (Evangelink) [inline src/Platform/Microsoft.Testing.Platform/IPC/NamedPipeBase.cs]: Add this string into a resx.

**Review by MarcoRossignoli — APPROVED** [review]

**Comment by Evangelink** [comment]
As discussed offline, for some technical reasons, we need to force merge this PR.

**Pull request merged**